Source/BLE.Client/BLE.Client/ViewModels/DeviceListViewModel.cs

```diff
@@ -399,6 +399,10 @@ private bool CanConnectToPrevious()
                 {
                     _userDialogs.ShowLoading($"Connecting to {item.Name} ...");
                     await Adapter.ConnectToDeviceAsync(item.Device);
+
+                    var resultMTU = await item.Device.RequestMtuAsync(60);
+                    System.Diagnostics.Debug.WriteLine($"Requested MTU. Result is {resultMTU}");
+
```

Comment:
This will modify the sample app to always attempt to negotiate MTU... maybe we can make this configurable. We will add a ToDo